README.md

```diff
@@ -22,6 +22,7 @@ Then add **hubot-rate-limit** to your `external-scripts.json`:
 
 `HUBOT_RATE_LIMIT_NOTIFY_PERIOD` - default interval (seconds) between posting rate limiting messages in chat
 `HUBOT_RATE_LIMIT_CMD_PERIOD` - default interval (seconds) between each invocation of a specific listener
+`HUBOT_RATE_LIMIT_NOTIFY_MSG` - (Optional) Setting this environment variable to any value will supress rate limit exceeded feedback messages in chat.
```

Comment:
I think the variable name is a little misleading and the documentation doesn't match the code (code only sends notifications when the variable is set). Maybe `HUBOT_RATE_LIMIT_SILENT` and `if not process.env.HUBOT_RATE_LIMIT_SILENT?` instead?